TACC/Core-CMS-Resources
# UTRC-357/FP-1268: Remove UTRC v1 Styling

# To Do

- [x] Test on UTRC Pre-Prod **after** 2021-09-27 launch.

# Required By

- https://github.com/TACC/Core-CMS/pull/360
  The "parent" PR.

# Overview

Remove v1-v2 migration styles from UTRC.

# Issues

- [FP-1268](https://jira.tacc.utexas.edu/browse/FP-1268)

# Testing

> Remote Server: https://pprd.utrc.tacc.utexas.edu/
> Image Build: https://jenkins01.tacc.utexas.edu/job/Core_CMS/325/
> Deploy Job: https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/941/


1. Ensure `migrate.v1_v2.css` does not load on Publications and About pages.
2. Ensure page\* content headings (`<h1>`, etc.) font is Benton Sans (not Open Sans) on Publications and About pages.
3. Ensure page\* content "prose" (`<p>`, `<li>`, etc.) font is Benton Sans (not Roboto) on Publications and About pages.†

> \* The content beneath the header, not the header. The header uses Benton Sans (mostly) and Roboto (search).
> † Roboto _might_ be intended in the body content, but that is _very_ unlikely. If found, please report.
